test/number-literal-case.js

```diff
@@ -0,0 +1,75 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/number-literal-case';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	},
+	parserOptions: {
+		sourceType: 'module'
+	}
+});
+
+const error = {
+	ruleId: 'number-literal-case',
+	message: 'Invalid number literal casing'
+};
+
+ruleTester.run('number-literal-case', rule, {
+	valid: [
+		'const foo = 0xFF',
+		'const foo = 0b11',
+		'const foo = 0o10',
+		`const foo = '0Xff'`
```

Comment:
Ah, my bad! Didn't look closely enough. That's a good test to have, as you often get surprising results when you're writing tests like that. :+1: